celery_tasks/tasks.py

```diff
@@ -0,0 +1,21 @@
+import pendulum
+
+from django.db.models import Sum
+
+from api.models import Salary, SalaryCache
+from .celery_app import app
+
+
+@app.task
+def add_to_salary_cached(id):
+    qs = Salary.objects.filter(company_employee__employee__id=id) \
+        .filter(month__gte=pendulum.now().subtract(years=1)) \
+        .aggregate(salary=Sum('salary'))
+
+    try:
+        salary_cache = SalaryCache.objects.get(employee__id=id)
```

Comment:
1. again, this should include `year`.
1. hint: use smth like `create_or_update()` instead of multiline `get+create`